index.css

```diff
@@ -25,424 +17,1056 @@ a {
   color: rgb(255, 255, 255);
 }
 
-.list-x a {
-  text-decoration: none;
-  border: solid #8b8282d3;
-  padding: 18px;
-  font-family: 'Inter', sans-serif;
-  font-size: 15px;
-  line-height: 20px;
-  font-weight: 500;
-  width: 43px;
-  height: 20px;
-  color: #fff;
-  background-color: rgba(255, 255, 255, 0.24);
-}
+@media (min-width: 375px) {
+  main {
+    display: flex;
+    flex-direction: column;
+    max-width: 768px;
+    min-width: 375px;
+    justify-content: space-around;
+    align-items: center;
+    margin: auto;
+    margin-top: 75px;
+  }
 
-.lists a {
-  text-decoration: none;
-  font-size: 22px;
-}
+  .list-x a {
+    text-decoration: none;
+    border: solid #8b8282d3;
+    padding: 18px;
+    font-family: 'Inter', sans-serif;
+    font-size: 15px;
+    line-height: 20px;
+    font-weight: 500;
+    width: 43px;
+    height: 20px;
+    color: #fff;
+    background-color: rgba(255, 255, 255, 0.24);
+  }
 
-.works-centered a {
-  text-decoration: none;
-  border: solid #8b8282d3;
-  padding: 500px, 100px, 100px, 100px;
-  font-family: 'Inter', sans-serif;
-  font-size: 15px;
-  margin: 13px;
-  color: #1c1a19;
-}
+  .lists a {
+    text-decoration: none;
+    font-size: 22px;
+  }
 
-.header a {
-  font-family: 'Roboto', sans-serif;
-  color: #fff;
-  text-decoration: none;
-  font-size: 18px;
-  margin-left: 20px;
-}
+  .works-centered a {
+    text-decoration: none;
+    border: solid #8b8282d3;
+    padding: 500px, 100px, 100px, 100px;
+    font-family: 'Inter', sans-serif;
+    font-size: 15px;
+    margin: 13px;
+    color: #1c1a19;
+  }
 
-.my-logo:hover,
-.lists a:hover {
-  color: #ff6b00;
-}
+  .header a {
+    font-family: 'Roboto', sans-serif;
+    color: #fff;
+    text-decoration: none;
+    font-size: 18px;
+    margin-left: 20px;
+  }
 
-.print-data a:hover {
-  background-color: rgba(255, 107, 0, 0.5);
-}
+  .my-logo:hover,
+  .lists a:hover {
+    color: #ff6b00;
+  }
 
-.project-list a:hover {
-  background-color: #ff6b00;
-}
+  .print-data a:hover {
+    background-color: rgba(255, 107, 0, 0.5);
+  }
 
-.header {
-  overflow: hidden;
-  position: fixed;
-  top: 0;
-  width: 100%;
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: space-between;
-  background-color: #3c3a39;
-}
+  .project-list a:hover {
+    background-color: #ff6b00;
+  }
 
-.header img {
-  margin-right: 20px;
-  cursor: pointer;
-  width: 30px;
-  height: 20px;
-}
+  .header {
+    overflow: hidden;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    background-color: #3c3a39;
+    margin-left: 0;
+  }
```

Comment:
- [ ] I have noticed that there is a spacing in the left side of the `header`. Kindly make sure that this spacing is removed to ensure that your portifolio has good UI and hence professional ☺️ 

Your design 😎 
![image](https://github.com/anyars-encarta/Porfolio/assets/99394374/3220321b-906a-46c8-a4f7-6354b111f600)

Figma design 😎 
![image](https://github.com/anyars-encarta/Porfolio/assets/99394374/b851a480-37b9-4ef6-a64f-3050c8472b9f)